src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -5369,6 +5369,79 @@ public override string ToString()
                 return sb.ToString();
             }
 
+            /// <summary>
```

Comment:
This belongs in TensorExtensionMethods, I think.

Reply:
Another possibility is to add a 'style' argument to the verbose ToString() that currently takes a boolean.

Instead of
```C#
public string ToString(bool withData, string fltFormat = "g5", int width = 100, CultureInfo? cultureInfo = null)
```

it would be something like this:

```C#
public enum TensorStringStyle { Metadata, Julia, Numpy }

public string ToString(TensorStringStyle style, string fltFormat = "g5", int width = 100, CultureInfo? cultureInfo = null) 
{
    if (style == TensorStringStyle.Metadata) return ToString();
    ...
}

// For backward compat:

public string ToString(bool withData, string fltFormat = "g5", int width = 100, CultureInfo? cultureInfo = null) 
{
    return ToString(withData ? TensorStringStyle.Julia : TensorStringStyle.Metadata, fltFormat, width, cultureInfo);
}
```

Reply:
Then how about this?
```cs
public static TensorStringStyle DefaultOutputStyle = TensorStringStyle.Metadata;

            public override string ToString() => ToString(DefaultOutputStyle);

            public string ToString(TensorStringStyle style, string fltFormat = "g5", int width = 100,
                CultureInfo? cultureInfo = null) => style switch {
                    TensorStringStyle.Metadata => ToMetadataString(),
                    TensorStringStyle.Julia => ToJuliaString(fltFormat, width, cultureInfo),
                    TensorStringStyle.Numpy => ToNumpyString(this, ndim),
                    _ => throw new InvalidEnumArgumentException("Not supported type")
                };
```

Reply:
Sure. Don't you want CultureInfo and fltFormat on the NumPy string?